pkg/daemon/daemon.go

```diff
@@ -2208,6 +2216,7 @@ func (dn *Daemon) updateConfigAndState(state *stateAndConfigs) (bool, bool, erro
 	if inDesiredConfig {
 		// Great, we've successfully rebooted for the desired config,
 		// let's mark it done!
+		klog.Infof("In desired state %s", state.getCurrentName())
```

Comment:
Curious why this was moved, wouldn't we want it to process either the Completing update or No update required lines first?

Alternatively, wouldn't those lines already cover the `In desired state` part so maybe this line is no longer necessary?

Reply:
I moved the line earlier in the block to convey that the system is in the desired state - this is recorded before any decision about whether an update is necessary (at least how i interpreted it). but I see what you're saying also and it can be confusing, i can remove or reword.